tensorbay/cli/auth.py

```diff
@@ -57,8 +58,9 @@ def _implement_auth(  # pylint: disable=too-many-arguments
     else:
         error(f'Invalid argument "{arg1}"')
 
+    _check_auth(arg1, arg2, obj["profile_name"])
```

Comment:
check auth needs to open config file
update_profile also needs to open config file
can we open the config only once?